app.js

```diff
@@ -7,5 +9,19 @@ const signUpEmail = document.getElementById('sign-up-email');
 const signUpPassword = document.getElementById('sign-up-password');
 
 // Wire up sign in and sign up forms to supabase
-// Redirect to /other-page on successful auth
-// Redirect to /other-page when page loads if user is authenticated
+// redirectIfLoggedIn();
+
+signUpForm.addEventListener('submit', async (e) => {
+    e.preventDefault();
+    console.log(signUpEmail.value, signUpPassword.value);
+    await signupUser(signUpEmail.value, signUpPassword.value);
+    redirectIfLoggedIn();
+});
+
+signInForm.addEventListener('submit', async (e) => {
+    e.preventDefault();
+    console.log(signInEmail.value, signInPassword.value);
```

Comment:
Great Job Kamie one thing I would go through and do is if you hit the lightbulb 💡 on the error it will ask you if you want to disable console.log for that line or file I would add those on the files you want to leave the logs. I do this all the time cause I love to validate.